### PR TITLE
[WP8] Updating docs and fixing the first registeration issue

### PR DESCRIPTION
**Pull request by rakatyal**

### CODE REVIEW

File: src/wp8/PushPlugin.cs [hunk 1]
```diff
@@ -29,10 +29,14 @@ public void register(string options)
...
             {
                 pushChannel = new HttpNotificationChannel(this.pushOptions.ChannelName);
-
+                SubscribePushChannelEvents(pushChannel);
                 try
                 {
-                    pushChannel.Open();
+                    var count = 0;
+                    while(count < 3 && pushChannel.ChannelUri == null)
+                    {
```
Comment (nikhilkh) [inline src/wp8/PushPlugin.cs]: Looks like you forgot to increment the count. :boom:

**Comment by nikhilkh** [comment]
It's a good idea to add a [WP8] to the commit message so it's clear what platform the commit is addressing.

File: src/wp8/PushPlugin.cs (hunk 1 above)
Comment (rakatyal) [inline src/wp8/PushPlugin.cs]: Oops! My bad. Corrected.

**Comment by nikhilkh** [comment]
LGTM. Please squash and consider updating title of the commit

**Pull request merged**